packages/visualizations/src/components/Table/TableCard.svelte

```diff
@@ -41,10 +41,20 @@
 </Card>
 
 <style>
+    :global(.ods-dataviz--default) .table-container {
+        border: solid 1px var(--border-color);
+        border-radius: var(--border-radius-6);
+        margin-bottom: var(--spacing-100);
+        overflow: hidden;
+        background-color: var(--background-color);
+    }
     :global(.ods-dataviz--default) div {
         max-width: 100%;
         display: flex;
         flex-direction: column;
         justify-content: space-between;
     }
+    :global(.table-container .pagination) {
```

Comment:
Has to be done Pagination, else .pagination isn't scoped and risks concflicts.

Reply:
I think we've made assumption on how Svelte scopes CSS

A class name is not transformed.

<img width="1224" alt="image" src="https://github.com/opendatasoft/ods-dataviz-sdk/assets/117300300/fe0e9417-3029-486a-9ea5-1157e12018a9">

In this example, the HTML `h1` will have a class attribute with a value: `'title'` 
And the style is only applied for this component tag as Svelte transforms the `h1` selector into a scoped selector`.title.svelte-zxiuva`

So IMO my point (https://github.com/opendatasoft/ods-dataviz-sdk/pull/253#discussion_r1663707612) stands. 
We should do a full rework on classes we want to expose to end users